docs/community/bindings.md

```diff
@@ -2,6 +2,10 @@
 
 As a C library, bindings can be made to call H3 functions from different programming languages. This page lists different bindings currently available. Contributions to this list are welcome, please feel free to open a [pull request](https://github.com/uber/h3/tree/master/docs/community/bindings.md).
 
+## Java
+
+- [uber/h3-java](https://github.com/uber/h3-java)
```

Comment:
Worth differentiating between "officially supported" bindings from Uber and "community" bindings from others?

Reply:
Nah, let the market decide. ;) The `uber` in front of this one will add some obvious bias, right?